### DISCUSSION TITLE
数据集合归档操作

### DISCUSSION THREAD

**Discussion by leatrise** [shion-app/shion#64] [Ideas]
计划是有时效性的，比如我这会需要考个四六级，但是当我考过了以后，那这个标签/计划就不会再被使用了，但是他又存在了很多记录，不想删除，但是放在这每次作为一个候选项都挺碍事的。
考虑一下做个“归档”（Archive）把已过时效的各种数据集合给隐藏起来呢

**Comment by hanaTsuk1** [comment]
> 每次作为一个候选项都挺碍事的

具体是什么场景？

> 把已过时效的各种数据集合给隐藏起来

可以细致描绘一下隐藏具体对哪些地方生效

因为隐藏掉了，代表着入口都没了，总不能唯一它存在过的痕迹只留在计时总和里吧
如果你单纯不想让这些项目占据你的视野或想给个标记，可以选择
+ 使用拖拽，拖到同类别队伍末尾
+ 创建一个归档的维度，打上标记

**Reply by leatrise** [reply]
![image](https://github.com/user-attachments/assets/846b8983-5e12-44f1-9931-4942dca5b56c)
就真比如说我这些课程，下学期就不会再用了，我就想着左下角这边有个归档的项目，里面也是这样的一个布局，就保留他的父级计划关系。
其实是我这一类全部都是放我的课的，那随着时间的推移我这个计划里面就会越来越多的标签，但是这些标签也仅仅是当前学期中生效，到了下个学期我就不会再使用了，所以这个使用场景下确实是
> 入口都没了，唯一它存在过的痕迹只留在计时总和里

**Reply by hanaTsuk1** [reply]
👌加入开发计划中
